### PR TITLE
Executor ITF: transfer add_asset_qty_test

### PR DESCRIPTION
**Pull request by MBoldyrev**
Signed-off-by: Mikhail Boldyrev <miboldyrev@gmail.com>

### Description of the Change

transfered add_asset_qty_test from postgres_executor_test & BIG ITF acceptance to new shiny Executor ITF.

### CODE REVIEW

File: test/integration/executor/add_asset_qty_test.cpp [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+  getItf().createDomain(kSecondDomain);
+  createAsset(kAssetName, kSecondDomain, 1);
+  assertResultValue(getItf().createUserWithPerms(
+      kUser, kDomain, kUserKeypair.publicKey(), {Role::kAddAssetQty}));
+
+  expectResultValue(getItf().executeCommandAsAccount(
+      *getItf().getMockCommandFactory()->constructAddAssetQuantity(
+          kAnotherDomainAssetId, kAmount),
+      kUserId,
+      true));
```
Comment (lebdron) [inline test/integration/executor/add_asset_qty_test.cpp]: `true` looks confusing to me, and I am not sure whether we need to execute any command without permission checks. Maybe add a TODO to make a more clear interface?

File: test/framework/common_constants.hpp [hunk 2]
```diff
@@ -40,6 +40,7 @@ namespace common_constants {
   extern const std::string kSameDomainUserId;
   extern const std::string kAnotherDomainUserId;
   extern const std::string kAssetId;
+  extern const std::string kAnotherDomainAssetId;
```
Comment (lebdron) [inline test/framework/common_constants.hpp]:
```suggestion
  extern const std::string kSecondDomainAssetId;
```
🤷‍♂

File: test/integration/executor/add_asset_qty_test.cpp [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+/**
+ * @given a user with kAddDomainAssetQty permission and an asset in another
+ * domain
+ * @when execute AddAssetQuantity command from that user for that asset
+ * @then the command fails
+ * @and the asset is not added to the user
+ */
+TEST_P(AddAssetQuantityBasicTest, DomainPermInvalidValid) {
```
Comment (lebdron) [inline test/integration/executor/add_asset_qty_test.cpp]:
```suggestion
TEST_P(AddAssetQuantityBasicTest, DomainPermInvalid) {
```

File: test/integration/executor/add_asset_qty_test.cpp (hunk 1 above)
Comment (MBoldyrev) [reply test/integration/executor/add_asset_qty_test.cpp]: I think we should only validate the commands that are part of test fulfilment criteria. The rest can go without validation if it makes them simpler.

**Pull request merged**